src/main/java/me/hypherionmc/sdlinklib/discord/commands/UnLinkCommand.java

```diff
@@ -63,16 +63,14 @@ protected void execute(CommandEvent event) {
         } else {
             tables.forEach(SQLiteTable::delete);
 
-            String nickName = (event.getMember().getNickname() == null || event.getMember().getNickname().isEmpty()) ? event.getAuthor().getName() : event.getMember().getNickname();
-            if (pattern.matcher(nickName).matches()) {
-                nickName = pattern.matcher(nickName).replaceAll("");
-            }
-
-            try {
-                event.getMember().modifyNickname(nickName).queue();
-            } catch (Exception e) {
-                if (modConfig.generalConfig.debugging) {
-                    e.printStackTrace();
+            String nickName = event.getMember().getEffectiveName();
+            if (pattern.matcher(nickName).find()) {
+                try {
+                    event.getMember().modifyNickname(null).queue();
```

Comment:
Same as before, don't clear the nickname

Reply:
Same as before